backend/functions/schemes/schemes.py

```diff
@@ -0,0 +1,66 @@
+"""
+url for local testing:
+http://127.0.0.1:5001/schemessg-v3-dev/asia-southeast1/schemes
+"""
+
+import json
+
+from fb_manager.firebaseManager import FirebaseManager
+from firebase_functions import https_fn
+from loguru import logger
+
+
+firebase_manager = FirebaseManager()
+
+
+@https_fn.on_request(region="asia-southeast1")
+def schemes(req: https_fn.Request) -> https_fn.Response:
+    """
+    Handler for single schemes page endpoint
+
+    Args:
+        req (https_fn.Request): request sent from client
+
+    Returns:
+        https_fn.Response: response sent to client
+    """
+
+    global firebase_manager
```

Comment:
generally not a good practice to use global variables because it will lead to testing and code readability issues, as it is not always clear where and how the global state is being modified. People typically use dependency injection or factory functions.

Can try to use a factory function